template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/impala/ImpalaLdapConfigProvider.java

```diff
@@ -0,0 +1,92 @@
+package com.sequenceiq.cloudbreak.cmtemplate.configproviders.impala;
+
+import static com.sequenceiq.cloudbreak.cmtemplate.configproviders.ConfigUtils.config;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.stereotype.Component;
+
+import com.cloudera.api.swagger.model.ApiClusterTemplateConfig;
+import com.cloudera.api.swagger.model.ApiClusterTemplateVariable;
+import com.sequenceiq.cloudbreak.cmtemplate.CmTemplateProcessor;
+import com.sequenceiq.cloudbreak.cmtemplate.configproviders.AbstractRoleConfigProvider;
+import com.sequenceiq.cloudbreak.dto.KerberosConfig;
+import com.sequenceiq.cloudbreak.dto.LdapView;
+import com.sequenceiq.cloudbreak.template.TemplatePreparationObject;
+import com.sequenceiq.cloudbreak.type.KerberosType;
+
+@Component
+public class ImpalaLdapConfigProvider extends AbstractRoleConfigProvider {
+
+    private static final String IMPALA_IMPALA_LDAP_URI = "impala-impala_ldap_uri";
+
+    private static final String IMPALA_LDAP_BASE_DN = "impala-ldap_baseDN";
+
+    private static final String IMPALA_LDAP_DOMAIN = "impala-ldap_domain";
+
+    @Override
+    public List<ApiClusterTemplateConfig> getServiceConfigs(CmTemplateProcessor templateProcessor, TemplatePreparationObject source) {
+        List<ApiClusterTemplateConfig> result = new ArrayList<>();
+        result.add(new ApiClusterTemplateConfig().name("enable_ldap_auth").value(Boolean.TRUE.toString()));
```

Comment:
Not a big deal, but now these could be simplified by using `config()` as in:

https://github.com/hortonworks/cloudbreak/blob/7e1c4245c291c184ec88ac9688eef70620b404a8/template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/impala/ImpalaLdapConfigProvider.java#L82

Reply:
Thanks, fixed.